src/utils/misc/readableTime.js

```diff
@@ -1,13 +1,21 @@
-function timeToHumanReadable(timeInSeconds) {
+function timeToHumanReadable(timeInSeconds, precision = "seconds") {
   if (timeInSeconds < 60) {
-    return timeInSeconds + (timeInSeconds > 1 ? " seconds" : " second");
+    if (precision === "seconds")
```

Comment:
maybe add a little comment saying that the other expected value of precision is "minutes"? otherwise it's a bit funny that one never knows any other possible value of precision.